src/java/picard/analysis/CollectRnaSeqMetrics.java

```diff
@@ -49,12 +49,52 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = "Collect metrics about the alignment of RNA to various functional classes of loci in the genome:" +
-                "coding, intronic, UTR, intergenic, ribosomal. Also determines strand-specificity for strand-specific libraries.",
-        usageShort = "Produces RNA alignment metrics for a SAM or BAM file",
+        usage = CollectRnaSeqMetrics.USAGE_SUMMARY + CollectRnaSeqMetrics.USAGE_DETAILS,
+        usageShort = CollectRnaSeqMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectRnaSeqMetrics extends SinglePassSamProgram {
+
+    static final String USAGE_SUMMARY = "Produces RNA alignment metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "<p>This tool takes a SAM/BAM file containing aligned RNA transcripts and produces metrics describing the distribution " +
+            "of the bases within the transcripts.  Calculates the total numbers and the percentages of nucleotides within the transcript sub-domains" +
+            " including the untranslated region (UTR), introns, intergenic sequences (between discrete genes) and peptide-coding sequences.  " +
+            "This tool also determines the numbers of bases that pass the sequencing vendor's quality filters (PF_BASES). " +
+            "To learn more about the PF metric, please see the corresponding GATK " +
+            "<a href='https://www.broadinstitute.org/gatk/guide/article?id=6329'><b>dictionary entry</b></a>.</p>" +
```

Comment:
OK.  I only did it because it seems difficult to notice the hypertext without bolding.  However, I take your point about consistency.  Will go back and remove the bold markers.